src/test/java/hw1/TestAddingReturnsCorrectSum.java

```diff
@@ -11,6 +12,7 @@
 
     private Calculator calc;
 
+    // TODO What is the purpose of these fields?
```

Comment:
I use them to compare actual result to expected. It is easier than specifying values within assertion. And I think it is also of better readability.